lua/translations/russian.lua

```diff
@@ -240,74 +240,74 @@ return {
-["waiting 10 more seconds... and then! i promise!"] = "Подождите еще 10 секунд, и после! Я обещаю!",
+["damage mode set to "] = "Режим урона изменен на ",
+["waiting 10 more seconds... and then! i promise!"] = "Подождите еще 10 секунд, и всё будет! Я обещаю!",
 ["cinematic"] = "Кинематографический",
-["play chatsounds"] = "Проиграть Звук Чата",
-["automatic jumping"] = "автоматический прыжок",
+["play chatsounds"] = "Проиграть звук чата",
+["automatic jumping"] = "Авто-Прыжок",
 ["copy name"] = "Скопировать имя",
 ["client"] = "Клиент",
-["server 1 is very slow, might take 2 minutes to boot back up..."] = "1 сервер очень медленный, может занять более 2 минут чтобы восстановить его...",
+["server 1 is very slow, might take 2 minutes to boot back up..."] = "1 сервер очень медленный. Потребуется 2 минуты чтобы восстановить его.",
 ["precache avatars"] = "Предварительное кэширование аватарок игроков",
 ["player volume"] = "Громкость медиа-плеера",
-["prop spam"] = "Спам пропами",
-["ungag"] = "Убрать кляп",
-["catch"] = "пойман",
+["prop spam"] = "Проп-Спам",
+["ungag"] = "Убрать заглушку",
+["catch"] = "Пойман",
 ["copy user id"] = "Копировать ID игрока",
 ["3d chat"] = "3D Чат",
 ["hurry!"] = "поторопись!",
 ["save"] = "сохранить",
-["shared"] = "общий",
+["shared"] = "Shared",
 ["panel size"] = "Размер панели",
 ["open"] = "Открыть",
 ["file"] = "файл",
-["member"] = "участник",
+["member"] = "Участник",
 ["check for updates"] = "Проверить обновления",
 ["copy player"] = "Копировать ID Игрока",
-["lua mode"] = "режим lua",
-["reconnecting soon..."] = "Переподключение скоро....",
+["lua mode"] = "Режим LUA",
+["reconnecting soon..."] = "Переподключение скоро..",
 ["me"] = "Себе",
-["fix"] = "Исправить",
-["bring"] = "телепорт",
+["fix"] = "Фикс",
+["bring"] = "к себе",
 ["enabled"] = "включен",
-["gave up"] = "сдался",
-["view achievements"] = "Посмотреть Достижения",
+["gave up"] = "потерял",
+["view achievements"] = "Посмотреть достижения",
 ["spawn"] = "создать",
 ["it sure is slow today :((("] = "Что-то он медленный сегодня :(((",
-["team chat -> pm"] = "командый чат -> ЛС",
+["team chat -> pm"] = "Чат команды -> ЛС",
 ["> green text"] = "> зелёный текст",
-["highlight messages including your nickname"] = "подсвечивать сообщения упоминающие ваш никнэйм",
+["highlight messages including your nickname"] = "Подсвечивать сообщения в которых упоминается твой ник",
 ["hide player"] = "Скрыть игрока",
 ["commited suicide."] = "покончил с собой.",
 ["hate everyone"] = "ненавидит всех",
 ["the server"] = "c сервера",
 ["restore"] = "Восстановить",
-["friend"] = "Настройки друзей(Общий доступ)",
+["friend"] = "Настройки друзей (общий доступ)",
 ["logging"] = "Регистрация",
 ["time"] = "время",
-["chat sounds"] = "Звуки чата",
+["chat sounds"] = "Чатовые звуки",
 ["are you sure that's everything?!"] = "Вы уверены, что это все?!",
-["you forgot something!"] = "Вы что-то забыли!",
+["you forgot something!"] = "Ты что-то забыл!",
```

Comment:
(on russian, sorry) Почему ты обращаешься к игроку на ТЫ, а не на ВЫ? (it's kinda hard to translate my message on english, because "ТЫ" and "ВЫ" on english are same, but not on russian)